### DISCUSSION TITLE
New Translations

### DISCUSSION THREAD

**Discussion by hborras** [kimai/kimai#5004] [Questions & Answers] [answered]
Hello all,

Just discovered this OS tool and looks amazing. 

I've tried to set Kimai in Catalan language but it is not available. So I've just decided maybe to try to do the translation by myself. 

I'm registered at weblate but I don't see the option to "request" a new language to start a translation .

Anyone knows how to do it?

Thanks

**Comment by kevinpapst** [comment, marked as answer]
Thats a nice offer, thanks 👍 

If you go into any component, e.g. https://hosted.weblate.org/projects/kimai/messages/

Than you should see a `+` in the upper left corner:
![image](https://github.com/user-attachments/assets/2acc653e-89c5-4af4-bc65-2748299244e9)

But I am not sure what follows next, I never started a new translation 😁

**Comment by hborras** [comment]
Wow, not too much intuitive, hahaha. 

Thanks!

**Comment by hborras** [comment]
Well, it "works" 

I can create a new translation, but i don't see how to start it. 

I will take a look next days

thanks
